src/language-css/utils.js

```diff
@@ -60,7 +60,7 @@ function getPropOfDeclNode(path) {
 
 function isSCSS(parser, text) {
   const hasExplicitParserChoice = parser === "less" || parser === "scss";
-  const IS_POSSIBLY_SCSS = /(\w\s*: [^}:]+|#){|@import[^\n]+(url|,)/;
+  const IS_POSSIBLY_SCSS = /(\w\s*: ?[^}:]+|#){|@import[^\n]+(url|,)/;
```

Comment:
Should it be `*`?

Reply:
I can't understand regexp too :smile: , I think we should rewrite it, on looking extension

Reply:
You mean file extension? File extension is not always available, maybe check actual parser?